src/assets/data/featured.json

```
@@ -0,0 +1,48 @@
+[
+  {
+    "daoName": "JankCryptoPioneers",
```

Let's just use `name`, otherwise we have to specify `dao` everywhere
```suggestion
    "name": "JankCryptoPioneers",
```